apps/server/src/shared/repo/user/user-do.repo.integration.spec.ts

```diff
@@ -441,4 +441,149 @@ describe('UserRepo', () => {
 			});
 		});
 	});
+
+	describe('find', () => {
+		const setupFind = async () => {
+			const query: UserQuery = {
+				schoolId: undefined,
+				isOutdated: undefined,
+				lastLoginSystemChangeGreaterThan: undefined,
+				outdatedSinceEquals: undefined,
+			};
+
+			const options: IFindOptions<UserDO> = {};
+
+			await em.nativeDelete(User, {});
+			await em.nativeDelete(School, {});
+
+			const userA: User = userFactory.buildWithId({ firstName: 'A' });
+			const userB: User = userFactory.buildWithId({ firstName: 'B' });
+			const userC: User = userFactory.buildWithId({ firstName: 'C' });
+			const users: User[] = [userA, userB, userC];
+			await em.persistAndFlush(users);
+
+			const emFindAndCountSpy = jest.spyOn(em, 'findAndCount');
+
+			return { query, options, users, emFindAndCountSpy };
+		};
+
+		describe('sorting', () => {
+			it('should create queryOrderMap with options.order', async () => {
+				const { query, options, emFindAndCountSpy } = await setupFind();
+				options.order = {
+					id: SortOrder.asc,
+				};
+
+				await repo.find(query, options);
+
+				expect(emFindAndCountSpy).toHaveBeenCalledWith(
+					User,
+					{},
+					expect.objectContaining<FindOptions<User>>({
+						orderBy: expect.objectContaining<QueryOrderMap<User>>({ _id: options.order.id }) as QueryOrderMap<User>,
```

Comment:
The as cast should not be needed anymore, after updating the jest version for some month.

Reply:
Nice thanks for the info.., i look forward to this :)